.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "vendor/bundle"
```

Comment:
🤔 this repo has no CI, and I don't have an Intel Mac with me to try, but I _think_ we should add:

```
BUNDLE_SPECIFIC_PLATFORM: "false"
```

To avoid potential issues with gems with native extensions and/or with the `Gemfile.lock` adding a new `PLATFORMS` entry for every kind of machine it runs on, e.g. [the WooCommerce iOS one](https://github.com/woocommerce/woocommerce-ios/blame/fe33de19c7a2f8507200cc6aa7ac7bc1b5a591e7/Gemfile.lock)

![image](https://user-images.githubusercontent.com/1218433/203534366-34e2bb57-9225-45ca-88a0-2faf6c171592.png)